src/content/blog/2025-10-03-whats-coming-in-eslint-10.0.0.md

```diff
@@ -0,0 +1,151 @@
+
+## Development plan
+
+Similar to v9.0.0, we will develop v10.0.0 in phases to ensure stability and gather community feedback:
+
+1. **Alpha.** The alpha release will include the most significant breaking changes that we expect will cause the most disruption for existing users. This early release will help us gather feedback and validate our approach.
+2. **Beta.** The beta release will include the remaining features and smaller breaking changes that will impact fewer users.
+
+After the beta has been validated through community testing, we will publish one or more release candidates as we continue to fix bugs and address compatibility issues.
+
+## Significant changes in v10.0.0-alpha
+
+The following changes are planned for the alpha release and represent significant breaking changes.
+
+### Dropping support for Node.js < v20.19.0
+
+Following our Node.js version support policy for major releases, ESLint v10.0.0 will [require Node.js v20.19.0 or later](https://github.com/eslint/eslint/issues/19969). The new supported versions will be:
+
+* Node.js v20.19.0 and above (Maintenance LTS)
+* Node.js v22.13.0 and above (Active LTS)
+* Node.js v24 and above (Current)
+
+This change aligns with Node.js's release schedule as of 2025, where:
+
+* Node.js v18 reached end-of-life in April 2025
+* Node.js v20 is in Maintenance LTS until April 2026
+* Node.js v22 became Active LTS in October 2024
+
+The updated Node.js requirements will enable ESLint to leverage newer JavaScript features, including native support for `require(esm)` (available since Node.js v20.19.0), and improved performance characteristics of modern Node.js versions.
+
+### Removal of eslintrc functionality
+
+In ESLint v9.0.0, we deprecated the old eslintrc config system while leaving the functionality available for backwards compatibility. In v10.0.0, we are [removing the eslintrc config system](https://eslint.org/blog/2023/10/flat-config-rollout-plans/#eslintrc-removed-in-eslint-v10.0.0) completely. Specifically, this means:
+
+1. The `ESLINT_USE_FLAT_CONFIG` environment variable is no longer honored.
+1. The CLI no longer supports eslintrc-specific arguments (`--no-eslintrc`, `--env`, `--resolve-plugins-relative-to`, `--rulesdir`, `--ignore-path`).
+1. `.eslintrc.*` and `.eslintignore` files will no longer be honored.
+1. The `shouldUseFlatConfig()` method will always return `true`.
+1. The `loadESLint()` function now always returns the `ESLint` class.
+1. The `Linter` constructor `configType` argument can only be `"flat"` and will throw an error if `"eslintrc"` is passed.
+1. The following `Linter` eslintrc-specific methods are removed:
+    * `defineParser()`
+    * `defineRule()`
+    * `defineRules()`
+    * `getRules()`
+1. The following changes to the `/use-at-your-own-risk` entrypoint:
+    * `LegacyESLint` is removed
+    * `FileEnumerator` is removed
+    * `shouldUseFlatConfig()` method will always return `true`
```

Comment:
Maybe "function" fits better here as this API isn't called on a specific object.

```suggestion
    * `shouldUseFlatConfig()` function will always return `true`
```

Also, the previous mention seems redundant?

https://github.com/eslint/eslint.org/blob/2d98ba6ac170a1c01c71ab3cfa7b52d440131844/src/content/blog/2025-10-03-whats-coming-in-eslint-10.0.0.md?plain=1#L52